### DISCUSSION TITLE
&SPEC line settings

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12099] [General]
Hi all,

I was recently running a sensitivity case between versions 6.7.7 and 6.8.0 to ensure that I got similar results and realized that I was not able to set up my burner fire in the same manner. The fire is a liquefied natural gas (methane) fire. This is how I have set it up historically: 

&SPEC ID='REAC_FUEL', FORMULA='C1H4'/

&REAC ID='METHANE',
      FUEL='REAC_FUEL',
      SOOT_YIELD=0.01,
      HEAT_OF_COMBUSTION=5.0E+4,
      RADIATIVE_FRACTION=0.25/

&SURF ID='Fire',
      COLOR='RED',
      MLRPUA=0.11/
      
      
However, I get the error in the command line that " SPEC REAC_FUEL is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen. "

Instead, I am planning to set up the fires like this: 

&SPEC ID='METHANE', FORMULA='C1H4'/

&REAC ID='METHANE',
      FUEL='METHANE',
      SOOT_YIELD=0.01,
      HEAT_OF_COMBUSTION=5.0E+4,
      RADIATIVE_FRACTION=0.25/

&SURF ID='Fire',
      COLOR='RED',
      MLRPUA=0.11/
      
If I'm understanding the technical manual correctly, this is the way to set it up. But I was hoping to get some confirmation before doing a bunch of runs. 

Thanks!

**Comment by drjfloyd** [comment]
It has always been the case that for a non-predefined species (e.g. REAC_FUEL) that any species properties not explicitly defined would use N2 Lennard-Jones values and assume a specific heat based on gamma=1.4. For small MW species (like C1H4 with an MW of 16) this was rarely an issue, but for large MW species the assumption of gamma=1.4 could cause problems. Though we have long had a warningabout this in the Guide, few users noticed it and we had many posts and issues related to users having high MW fuels without defining properties. In the hopes of reducing that we also added a warning when running FDS. This is not an error message, it is a warning that you should make sure your inputs are appropriate for you specific needs.Since METHANE is predefined, you would be better off just having:

&REAC FUEL='METHANE', SOOT_YIELD=0.01/

FDS has a predefined radiative fraction for METHANE. FDS also knows its heat of formation, and thus, can caculate the heat of combustion.

**Reply by Tbuttts21** [reply]
Thank you for the input drjfloyd. 

I will start using &REAC FUEL='METHANE', SOOT_YIELD=0.01/ from now on. 

It seems like the way I was setting it up wasn't necessarily 'invalid', just a roundabout way of doing it. Right?

**Comment by drjfloyd** [comment]
REAC_FUEL wasn't the best approach as you weren't using methane physical properties for specific heat, conductivity, etc.

The other approach is roughly equivalent. The heat of combustion will be slightly different (FDS will compute using heats of formation and account for minor product yields) and FDS assumes a 20 % radiative fraction for methane.

**Reply by Tbuttts21** [reply]
Okay thank you again for the input. Much appreciated.
